src/Apps/Order/orderRoutes.tsx

```diff
@@ -136,7 +137,11 @@ export const orderRoutes: AppRouteConfig[] = [
         }
       }
 
-      return <Component {...props} />
+      return (
+        <PaymentProvider>
+          <Component {...props} />
+        </PaymentProvider>
+      )
```

Comment:
@damassi I'd like to get your feedback especially on this part. it kinda feels off to provide the wrapper in the file where routes are defined. 

do you have any feedback?

Reply:
This feels right! The idea here is to incrementally adopt and expand context state over time. So you have a context for the payment portion of the app -- it should only wrap that part of the route, since it's the root. Then later, if state expands, you wrap another route root with another context, then another. And finally, as the final refactor, you can combine all of the state together into one `OrderContext` and merge each of the files together into one single context and then access slices of the state via a single context hook. But this isn't strictly necessary, and incrementally adopting by opting in to certain routes the way you're doing here is def the way to go. Each route is effectively its own sub-app.

Reply:
thanks, @damassi!